readme.md

```diff
@@ -1,7 +1,7 @@
 # Product cards
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://Rusyn-Roman.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://Rusyn-Roman.github.io/layout_product-cards/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="945" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/3a1ffb79-e4da-4b5e-959e-32c073df3a95">